frontend/src/components/Rides/RideProfile.js

```diff
@@ -210,12 +210,12 @@ class RideProfile extends Component {
         return (
             <Segment>
                 <Header>
```

Comment:
In semantic ui you can make an h5 like this, instead of using the HTML h5:
`<Header as="h5">`

Reply:
oh ya, however not sure what difference would it bring between using HTML h5 and Semantic UI header? Just wondering